Introductory/projects/EngineMonitor.json

```diff
@@ -8,25 +8,25 @@
   }, {
     "source" : "app/EngineMonitor",
     "target" : "type/com.vantiq.engines.SystemHUD"
-  }, {
-    "source" : "type/com.vantiq.engines.SystemHUD",
-    "target" : "subscription/SystemHUD"
   }, {
     "source" : "type/com.vantiq.engines.EngineTemp",
     "target" : "app/EngineMonitor"
   }, {
-    "source" : "client/EngineSimulation",
-    "target" : "topic//my/response/topic"
+    "source" : "type/com.vantiq.engines.SystemHUD",
+    "target" : "subscription/SystemHUD"
   }, {
     "source" : "type/com.vantiq.engines.SystemStatus",
     "target" : "client/EngineSimulation"
   }, {
     "source" : "type/com.vantiq.engines.SystemHUD",
     "target" : "client/EngineSimulation"
+  }, {
+    "source" : "client/EngineSimulation",
+    "target" : "topic//my/response/topic"
   } ],
   "name" : "EngineMonitor",
   "options" : {
-    "description" : "The EngineMonitor (Introductory) Tutorial walks through the creation of a simple app shown below.\n\nTo test the app out, start the event generator. When the event generator starts, you should see badges appear over each task indicating the execution count for each task.\n\nAs data flows through the app, you'll notice events show up in real time in the SystemHUD subscription to the right of the app. This shows the updates made to the HUD in real time. To view the raw output of any task in the app, right click on any of the tasks and select \"View Task Events\", which will open a subscription pane that shows each event as it flows out of the task.\n\nBelow the app you should see a client which visualizes different pieces of the simulation. When the event generator, app and client are all running you should see line charts in the client showing the independently changing speed and temperature readings, as well as guages showing the most recent values and a pie chart showing the ratios of the different alert messages.\n\nPause the client using the pause button in the title bar and click on any of the widgets or view the Data Streams to learn more about the client implementation. ",
+    "description" : "",
```

Comment:
project has lost the description, so please put that back.

Should also make the description one of the panes that is open when the project is opened.  
Same for Test Intro project.